components/figma/figma.app.mjs

```diff
@@ -1,11 +1,96 @@
+import axios from "axios";
```

Comment:
If actions are calling methods from the app file, we should use Pipedream's axios.
```suggestion
import { axios } from "@pipedream/platform";
```